src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteUploadCommand.java

```diff
@@ -61,6 +77,11 @@ protected void execute(final CommandMap commandMap,
         final Gson gson = new GsonBuilder().registerTypeAdapter(Task.class,
                 new TaskDeserializer(configuration.getProjectName())).create();
         final Configuration instructions = this.loadConfiguration(commandMap);
+        // Get the countries filter
+        final List<String> countries = (List<String>) commandMap.get(COUNTRIES);
```

Comment:
I believe you can use getOptional which will mean you don't have to worry about nulls, just for safety's sake.